build.sbt

```diff
@@ -11,6 +11,10 @@ val commonSettings = Seq(
     "-language:_",
     "-encoding", "UTF-8"
   ),
+  javacOptions ++= List(
+    "-Xlint:unchecked",
+    "-Xlint:deprecation"
```

Comment:
Does this mean the warnings are still shown when the generated Java code is compiled in, say, a Maven project? If so then it might be worth seeing if we can avoid them entirely instead of suppressing them here? Could be a subsequent issue though.

Reply:
This enabled printing them so I could figure out what was wrong. No suppression going on!

Reply:
Ah of course, excellent!